tmt/utils.py

```diff
@@ -4661,6 +4661,7 @@ def field(
         choices: Union[None, Sequence[str], Callable[[], Sequence[str]]] = None,
         multiple: bool = False,
         metavar: Optional[str] = None,
+        envvar: Optional[str] = None,
```

Comment:
Is the change to `tmt/utils.py` still needed?

Reply:
it shouldn't hurt and maybe someone will want to use it at some point?
for this patch it's not needed, might be better to remove